src/caret_analyze/architecture/struct/node.py

```diff
@@ -248,6 +248,28 @@ def assign_variable_passings(self, callback_name_write: str, callback_name_read:
                     for passing in self._variable_passings_info]:
             self._variable_passings_info.append(passing)
 
+    # def remove_message_context(self, node_name: str, context_type: str,
+    #                            sub_topic_name: str, pub_topic_name: str):
+    #     # To assign message context, update_node_path() is called in Architecture.
+    #     # This is because module dependency.
+    #     # TODO: Refactoring module dependency
+    #     pass
```

Comment:
Delete this unless there is a specific reason not to.

Reply:
Fix it in [165b4e8](https://github.com/tier4/CARET_analyze/pull/297/commits/165b4e8d6f20e698deebe5948d498fa8a9c79c06).